content/pages/2024/tickets.md

```diff
@@ -72,5 +72,20 @@ attend EuroSciPy:
 
 <br>
 
-### Ticket prices
-TBA
+
+
+### Get your ticket
+
+<html>
+  <head>
+    <script src="https://js.tito.io/v2" async></script>
+  </head>
+  <body>
+    <tito-widget
+      event="euroscipy/2024"
+    ></tito-widget>
+  </body>
+</html>
+
+
+In case that you have additional queries, please contact us at: <a href="mailto:info@euroscipy.org">info@euroscipy.org</a>.
```

Comment:
```suggestion
In case that you have additional queries, please contact us at: <a href="mailto:info@euroscipy.org">info@euroscipy.org</a>.

(*) After registration, please send your student qualifying proof to
<a href="mailto:students@euroscipy.org">students@euroscipy.org</a>
```